test/services/CalculationServiceSpec.scala

```diff
@@ -1060,7 +1133,70 @@ class CalculationServiceSpec extends UnitSpec {
       }
     }
   }
-    //###################### Zero gain tests #############################
+
+  "Calling CalculationService.calculateFlatPRR" should {
+
+    "return 0 for a Disposal Date of 06-10-2016 and no Acquisition Date" in {
+      val result = CalculationService.calculateFlatPRR(Some("2016-10-06"), None, Some(0), 1000)
+      result shouldEqual 0
+    }
+
+    "return 0 for a Disposal Date of 05-10-2016 and no Acquisition Date" in {
+      val result = CalculationService.calculateFlatPRR(Some("2016-10-05"), None, Some(0), 1000)
+      result shouldEqual 0
+    }
+
+    "return £2019 for a Disposal Date of 05-10-2016, Acquisition Date of 05-04-2015, Days Eligible of 5 and Gain of £2000" in {
+      val result = CalculationService.calculateFlatPRR(Some("2016-10-05"), Some("2015-04-05"), Some(5), 2000)
+      result shouldEqual 2019
```

Comment:
Mark's requirement states that:
- If PRR calculation result is greater than the gain then the PRR should be capped to equal the amount of the gain (PRR cannot create a Loss). 

Although I think this is catered for by subsequent logic in the calc I think it would be better if this function which "Calculates Flat PRR" specifically caps it at the gain if the PRR is greater than the Gain. It just means that this particular function is accurate.